src/cloud/components/ApplicationContent.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import {
+  ContentLayoutContent,
+  ContentLayoutContentProps,
+} from '../../design/components/templates/ContentLayout'
+
+const ApplicationContent: React.FC<ContentLayoutContentProps> = ({
```

Comment:
Made it for mostly two reasons: 

1. Keep the naming logic between `ApplicationPage, ApplicationContent etc...`
2. Extending this component with an initialized check to avoid displaying the flashing red alerts when refreshing.